### PR TITLE
#1857 Dependency Update

### PR DESCRIPTION
**Pull request by mchlwellman**
# Description

This PR Is a regular dependency update. There was one failure - protobuf, which is captured in [this ticket](https://github.com/department-of-veterans-affairs/notification-api/issues/1886).

issue #1857 

## Type of change

Please check the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (quick fix for an urgent bug or issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation changes only

## How Has This Been Tested?

- [Deployed to dev](https://github.com/department-of-veterans-affairs/notification-api/actions/runs/10112337144) with no regression failures

## Checklist

- [x] I have assigned myself to this PR
- [x] PR has an [appropriate title](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/team_agreement.md#naming-prs): #9999 - What the thing does
- [x] PR has a detailed description, including links to specific documentation
- [x] I have added the [appropriate labels](https://github.com/department-of-veterans-affairs/notification-api/blob/master/.github/release.yaml) to the PR.
- [x] I did not remove any parts of the template, such as checkboxes even if they are not used
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to any documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have ensured the latest master is merged into my branch and all checks are green prior to review
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] The ticket is now moved into the DEV test column
- [x] I have added a bullet for this work to the Engineering Key Wins slide for review

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -36,6 +36,7 @@ marshmallow-sqlalchemy = "*"
 nanoid = "*"
 notifications-python-client = "*"
 phonenumbers = "*"
+protobuf = "^4.25"  # TODO: Pinned because version 5.27.2 failed to deploy
```
Comment (EvanParish) [inline pyproject.toml:39]: Did you create a ticket to look into this issue? The ticket number should be in the TODO comment.

**Review by MackHalliday — DISMISSED** [review]
🚀

File: pyproject.toml (hunk 1 above)
Comment (cris-oddball) [reply pyproject.toml:39]: ^^^ please make that ticket and add the ticket number here

**Review by cris-oddball — CHANGES_REQUESTED** [review]
Please check the [makefile](https://github.com/department-of-veterans-affairs/notification-api/blob/b6fc687771c8758de8d4ed9a8e59c185f860900f/Makefile#L35). When jinja2 was updated it cleared the issue there, so the comment about jinja can be removed and the safety check [line](https://github.com/department-of-veterans-affairs/notification-api/blob/b6fc687771c8758de8d4ed9a8e59c185f860900f/Makefile#L37) can have 70612 removed. If makefile is not mentioned in the template, let me know and I'll update it.

**Review by cris-oddball — APPROVED** [review]

**Review by EvanParish — APPROVED** [review]

**Review by MackHalliday — APPROVED** [review]
🌈

**Pull request merged**